src/sonic-yang-models/yang-models/sonic-neigh.yang

```diff
@@ -11,6 +11,11 @@ module sonic-neigh {
 		prefix yang;
 	}
 
+	// TODO: Uncomment the following lines when sonic-vlan.yang is available
+	// import sonic-vlan {
```

Comment:
how did it work for original PR? I don't think we should have dead code. Yang sub-group to check.

Reply:
https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-vxlan.yang#L87-L90
We have other comments for back-links issue.

Reply:
I removed the import in the original PR, I'm just adding comments with import and ref so we can have the TODO for when we can enable the import. ask: https://github.com/sonic-net/sonic-buildimage/pull/14521#discussion_r1173194179